mesh/ncmesh.cpp

```diff
@@ -4307,6 +4307,20 @@ int NCMesh::GetElementDepth(int i) const
    return depth;
 }
 
+int NCMesh::GetElementSizeReduction(int i) const
+{
+   int elem = leaf_elements[i];
+   int reduction = 1, parent;
```

Comment:
minor: maybe more clear that the comma functions as a separator and not an operator here if reversed:
int parent, reduction = 1;

Reply:
Yes, I have addressed this now.